packages/commons/package.json

```diff
@@ -26,13 +26,17 @@
     "@zodios/express": "10.6.1",
     "axios": "1.7.1",
     "connection-string": "4.4.0",
+    "date-fns": "3.6.0",
+    "date-fns-tz": "^3.1.3",
+    "handlebars": "^4.7.8",
```

Comment:
Remove `^` to pin dependencies, we don't use `^` anymore now that renovate is set-up and running

Reply:
Done with commit [169abfc](https://github.com/pagopa/interop-be-monorepo/pull/520/commits/169abfcea2cfdd1497fece6b5eb7e256c582f3df)